### DISCUSSION TITLE
web design

### DISCUSSION THREAD

**Discussion by AishaKeyf** [dugsiiyeinc/dhaqsoPay#6] [Ideas]
mid kaan kalana waan soo helay. Dookhu waa xagiina.
hays martiyaynina!
ok.

![4e611f73ad50c2972e226a615013c64c](https://github.com/user-attachments/assets/fadd0f31-169a-486b-9e35-80472774e89a)

https://dribbble.com/shots/19202772-Bankai-Online-Payment-Landing-Page?utm_source=Clipboard_Shot&utm_campaign=rizkirmw&utm_content=Bankai+-+Online+Payment+Landing+Page&utm_medium=Social_Share&utm_source=Pinterest_Shot&utm_campaign=rizkirmw&utm_content=Bankai+-+Online+Payment+Landing+Page&utm_medium=Social_Share

**Comment by A-M-Ahmed** [comment]
cool, waa udad  badsnena.

**Reply by shiinedev** [reply]
Designka waa fadhiya lkn wa land page oo kaliya waxa kasii qurux badnan lahed mid pages kala duwan leh

**Reply by AishaKeyf** [reply]
pages dheeraad ah haddii loo baahdo waa ku darsan karnaa, kan kaleba waa kasoo goosan karnaa, deetana si jacburis ah baa isugu dhisaynaa.
say ila thy, sikastaba waan isla fiirinaynaa. i.a.

**Reply by shiinedev** [reply]
Cool wye markas

**Comment by sharafdin** [comment]
Eng kan waa hal page sido kalena waa template ku qurxoon mobile app landing page ahaan marka option kale aan raadino Insha'Allah!
